### DISCUSSION TITLE
"Failed to construct 'Worker'" after Chrome update

### DISCUSSION THREAD

**Discussion by tokarchyn** [webpack/webpack#16531] [Q&A]
Hi, guys!
I am trying to figure out why I get the error provided below after Chrome has been updated to version `108.0.5359.95`. I use webpack `5.75.0`, and in my code I instantiate Worker like this `const worker = new Worker(new URL('./worker.ts', import.meta.url));`. The first load ok, but then error starts to appear after each webpage reload. If I open in a new tab the first load again works ok. In Edge and Firefox this problem does not take place.

Error:
```
Uncaught DOMException: Failed to construct 'Worker': Script at 'chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/src_app_services_worker_ts.js' cannot be accessed from origin 'http://localhost:4200'.
```

I also noticed that on the first load generated URL is:
```
{
  host: 'localhost:4200';
  hostname: 'localhost';
  href: 'http://localhost:4200/32e45026bb126f24.ts';
  origin: 'http://localhost:4200';
  password: '';
  pathname: '/32e45026bb126f24.ts';
  port: '4200';
  protocol: 'http:';
}
```
And after second:
```
{
  host: 'fmkadmapgofadopljbjfkapdkoienihi';
  hostname: 'fmkadmapgofadopljbjfkapdkoienihi';
  href: 'chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/32e45026bb126f24.ts';
  origin: 'chrome-extension://fmkadmapgofadopljbjfkapdkoienihi';
  password: '';
  pathname: '/build/32e45026bb126f24.ts';
  port: '';
  protocol: 'chrome-extension:';
}
```

**Comment by alexander-akait** [comment]
Do you bundle chrome extension? Do you have any non official plugins or loader?
